src/main.rs

```diff
@@ -75,21 +73,33 @@ async fn main() -> std::io::Result<()> {
 
     let connection_pool = web::Data::new(create_db_pool());
     let secret_key = Key::generate();
-    HttpServer::new( move || {
+    HttpServer::new(move || {
         App::new()
             .wrap(IdentityMiddleware::default())
             .wrap(SessionMiddleware::new(
-                 RedisActorSessionStore::new(get_redis_uri()),
-                 secret_key.clone()
+                RedisActorSessionStore::new(get_redis_uri()),
+                secret_key.clone(),
             ))
             .app_data(connection_pool.clone())
-            .route("/user/register", web::post().to(routes::user::user_register))
+            .route(
+                "/user/register",
+                web::post().to(routes::user::user_register),
+            )
             .route("/user/login", web::post().to(routes::user::user_login))
             .route("/user/logout", web::post().to(routes::user::user_logout))
             .route("/user/update", web::put().to(routes::user::user_update))
             .route("/user/delete", web::delete().to(routes::user::user_delete))
             .route("/user/info", web::get().to(routes::user::user_info))
             .route("/user/list", web::get().to(routes::user::user_list))
+            .route(
+                "/region/create",
+                web::get().to(routes::region::region_create),
+            )
+            .route("/region/list", web::get().to(routes::region::region_list))
+            .route(
+                "/region/update",
+                web::get().to(routes::region::region_update),
+            )
```

Comment:
Shouldn't'd be this something like this?
```suggestion
            .route(
                "/region",
                web::post().to(routes::region::region_create),
            )
            .route("/region", web::get().to(routes::region::region_list))
            .route(
                "/region/:id", // i don't know how path params in artix work
                web::put().to(routes::region::region_update),
            )
```

Reply:
totally correct fixed it